debugger/src/main/java/org/codice/acdebugger/breakpoints/AccessControlContextInfo.java

```diff
@@ -199,25 +205,70 @@ public AccessControlContextInfo grant(String domain) {
     return new AccessControlContextInfo(this, domain);
   }
 
+  @SuppressWarnings("squid:S106" /* this is a console application */)
+  void dumpTroubleshootingInfo(boolean osgi) {
```

Comment:
ℹ️ Extracted from the `SecurityCheckInformation` class where it was done. That should have been moved here when I introduced this class to hold the access control context info.